docs/getting_started/security_considerations.md

```diff
@@ -0,0 +1,29 @@
+# Security considerations
+
+nodecg-io is managing all configuration values for your service instances which consists of authentication credentials for third party services.
+nodecg-io tries to protect this sensitive data but can't make it completely safe due to constraints in the NodeCG architecture.
+
+This document will explain what nodecg-io can and can't do to protect your sensitive configuration data including credentials.
+The first part consist of our security claims and the second part describes how achieve those claims.
+
+## Security claims
+
+These claims all assume that you use a good strong password and take care of it properly.
+
+1.   No service configuration is accessible to someone with only filesystem access.
+    -   A exception to this is a nodecg-io install with automatic login as the password is stored in plain text.
+2.   No bundle will be able to access your plain text password.
```

Comment:
This seems to contradict point 3. Did you mean not loaded bundle?

Reply:
The plain text password is only stored inside the nodecg-io dashboard and not transmitted to the backend and never enters NodeCG.
While writing that I thought this would mean the password is safe but other bundles can obviously also have dashboards, can use `window.parent` to escape the iframe and then interact with the iframe of our dashboard. From there the password is readable using the HTML input element. So this is sadly not true anymore.